src/about/repl/repl/host/window.js

```diff
@@ -70,7 +70,7 @@ const evalContext =
 
 export const evaluate =
   (id/*:ID*/, code/*:string*/)/*:Task<Never, EvaluationResult>*/ =>
-  Task.future(() => new Promise(resolve => {
+  new Task((succeed, fail) => void(new Promise((resolve, reject) => {
```

Comment:
Ah, I think I understand: otherwise a `Promise` would be returned which would cause the `Task` chain to be extended, right? I'm not sure I'm too big a fan of this pattern as it's quite subtle, but it does make sense.